inference-engine/tests/functional/plugin/cpu/single_layer_tests/loop.cpp

```diff
@@ -249,40 +242,51 @@ const std::vector<ElementType> inputPrecisions = {
         ElementType::i8
 };
 
-std::vector<bool> execute_first_iteration{true};
-std::vector<bool> is_body_condition_const{true};
-std::vector<bool> body_condition{true, false}; // works only if is_body_condition_const == true
-std::vector<int64_t> trip_count{1, 5};
-std::vector<InputShape> inputs = {
-        {{-1, 1, -1}, {{10, 1, 10}, {1, 1, 1}, {1, 1, 1}, {5, 1, 3}}},
-        {{-1, 1, -1}, {{1, 1, 1}, {5, 1, 2}, {5, 1, 2}, {5, 1, 3}}},
-        {{-1, 1, -1}, {{10, 1, 10}, {5, 1, 2}, {5, 1, 2}, {5, 1, 3}}}
+std::vector<InputLayerType> trip_count_type { InputLayerType::CONSTANT, InputLayerType::PARAMETER };
+std::vector<int64_t> trip_count { 1, 5 }; // works only if trip_count_type is constant
+
+std::vector<std::vector<InputShape>> inputs = {
```

Comment:
Could you please rearrange the input shapes description to distinguish the dynamic shape and target shapes as we usually do in other SL tests with dynamic part? Here and in the TI test.

Reply:
Done, thanks